spec/integration/admin/grove_monitor_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+include Pages::Authentication
+
+RSpec.feature 'Grove Monitor' do
+  let(:teacher) { create(:teacher) }
+  let(:grove_monitor_page) { Pages::GroveMonitorPage.new }
+  let(:dashboard_page) { Pages::DashboardPage.new }
+
+  describe 'as a teacher' do
+    before { login(teacher) }
+
+    it 'can be reached from the main dashboard' do
+      dashboard_page.click_on("Grove Monitor")
+      expect(current_path).to eq(admin_grove_monitor_path)
+    end
+
+    it 'provides navigation links' do
+      expect(grove_monitor_page).to have_content("Teacher Dashboard")
```

Comment:
I dont think this test is super valuable. as well as the other ones of this nature down below. I think its more important there is a test that goes through the whole workflow by clicking on the link and going to the next page.